exercises/concept/meltdown-mitigation/.docs/instructions.md

```diff
@@ -5,18 +5,18 @@ In this exercise, we'll develop a simple control system for a nuclear reactor.
 For a reactor to produce the power it must be in a state of _criticality_.
 If the reactor is in a state less than criticality, it can become damaged.
 If the reactor state goes beyond criticality, it can overload and result in a meltdown.
-We want to mitigte the chances of meltdown and correctly manage reactor state.
+We want to mitigate the chances of meltdown and correctly manage reactor state.
 
 The following three tasks are all related to writing code for maintaining ideal reactor state.
 
 ## 1. Check for criticality
 
-The first thing a control system has to do is check if the reactor is balanced in criticality.
+The first thing a control system has to do is checking if the reactor is balanced in criticality.
```

Comment:
```suggestion
The first thing a control system has to do is check if the reactor is balanced in criticality.
```

Reply:
This phrasing (as strange as it sounds) is an imperative (_has to do a check_ ) but maybe the whole thing should be rephrased into _`"Checking if the reactor is balanced in criticality is the first thing a control system has to do."`_  for clarity?